services/application_service.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationService
+  def self.call(*args)
+    new(*args).call
```

Comment:
That's cool but you've used `something.new(params).call` previously so basically this has no use :man_facepalming: